ui/helpers/constants/routes.ts

```diff
@@ -132,6 +133,10 @@ const PATH_NAME_MAP = {
   [CONFIRM_IMPORT_TOKEN_ROUTE]: 'Confirm Import Token Page',
   [CONFIRM_ADD_SUGGESTED_TOKEN_ROUTE]: 'Confirm Add Suggested Token Page',
   [NEW_ACCOUNT_ROUTE]: 'New Account Page',
+  ///: BEGIN:ONLY_INCLUDE_IN(build-flask)
+  [ADD_SNAP_ACCOUNT_ROUTE]: 'Add Snap Account List Page',
+  [`${ADD_SNAP_ACCOUNT_ROUTE}/:snapId`]: `Add Snap Account Page`,
+  ///: END:ONLY_INCLUDE_IN
```

Comment:
This code is inside the `build-flask` fence instead of the `snaps` one. Is that correct?

Reply:
moved back into snaps in 479f66fbe539993ca94c06c0519910f29cfe8ff2